lib/metadata.js

```diff
@@ -770,7 +793,10 @@ TableMetadata.prototype.build = function (tableRow, columnRows) {
       this.clusteringOrder.push(c.type.options.reversed ? 'DESC' : 'ASC');
     }
   }
-  if (!this.isCompact) {
+  if (typeof tableRow['is_dense'] !== 'undefined') {
+    this.isCompact = tableRow['is_dense'];
+  }
+  else {
```

Comment:
(Updated to reflect correct line)
Ah, I see the problem now, the code below says:  

>  also, compact pk, ck and val appear as is_dense false

So I think if we make this if(!this.isCompact) that should fix it.  What do you think?